src/posts/implementation/how2Fable.ipynb

```diff
@@ -0,0 +1,778 @@
+    "```\n",
+    "\n",
+    "```fsharp\n",
+    "/// App.fs\n",
+    "///\n",
+    "printfn $\"{SearchMe.iAmAVeryLongAndEasilySearchableName}\"\n",
+    "```\n",
+    "\n",
+    "`npm run build` will bundle all Fable-compiled JS files into `public\\bundle.js`. This file will contain imports from `./src/.fable/`, \n",
+    "but also one for each F# source file. If we search it for `iAmAVeryLongAndEasilySearchableName` or `drawOneAndPrintCard` we will find the respective JS code.\n",
+    "\n",
+    "<br>\n",
+    "\n",
+    "Because Fable can only compile code patterns and the functions from the modules in ``src\\.fable`` it is **not** possible to compile any F# library with Fable out of the box.\n",
+    "\n",
+    "If we add `<PackageReference Include=\"FSharp.Stats\" Version=\"0.4.3\" />` to `App.fsproj` it will return an error.\n",
+    "\n",
+    "```fsharp\n",
+    "/// App.fs\n",
+    "///\n",
+    "open FSharp.Stats\n",
+    "\n",
+    "// Example from https://fslab.org/FSharp.Stats/BasicStats.html\n",
+    "let mean1 = \n",
+    "    [10; 2; 19; 24; 6; 23; 47; 24; 54; 77;]\n",
+    "    |> Seq.meanBy float\n",
+    "\n",
+    "printfn $\"{mean1}\"\n",
+    "```\n",
+    "\n",
+    "Error in powershell/cmd.\n",
+    "```powershell\n",
+    "F# compilation finished in 59ms\n",
+    "C:/Users/User/source/repos/How2Fable/src/App.fs(70,13): (70,18) error FSHARP: The namespace 'Stats' is not defined. (code 39)\n",
+    "C:/Users/User/source/repos/How2Fable/src/App.fs(74,12): (74,18) error FSHARP: The value, constructor, namespace or type 'meanBy' is not defined. Maybe you want one of the following:\n",
+    "   maxBy\n",
+    "   minBy\n",
+    "   maxBy\n",
+    "   minBy (code 39)\n",
+    "Watching src\n",
+    "```\n",
+    "\n",
+    "Fable has no idea about the `Seq.meanBy` function and cannot translate it to JS. For a list of Fable compatible libraries you can look \n",
+    "[here](https://fable.io/docs/dotnet/compatibility.html) for F# Core libraries and [here](https://fable.io/resources.html#Libraries) for community ressources.\n",
+    "\n",
+    "It is also **possible to make F# libraries Fable compatible** as described [here](https://fable.io/docs/your-fable-project/author-a-fable-library.html). \n",
+    "This was for example done for the [ISADotNet library](https://github.com/nfdi4plants/ISADotNet/blob/developer/src/ISADotnet/ISADotNet.fsproj#L65) with a \n",
+    "conditional trigger, to publish two nuget package versions. One with and one without Fable compatibility.\n",
+    "\n",
+    "\n",
+    "<details>\n",
+    "<summary>App.fsproj code</summary>\n",
+    "\n",
+    "```xml\n",
+    "<!--App.fsproj-->\n",
+    "<ItemGroup Condition=\" '$(PackageId)' == 'ISADotNet.Fable' \">\n",
+    "  <Content Include=\"*.fsproj; **\\*.fs; **\\*.fsi\" PackagePath=\"fable\\\" />\n",
+    "</ItemGroup>\n",
+    "```\n",
+    "\n",
+    "</details>\n",
+    "\n",
+    "\n",
+    "<details>\n",
+    "<summary>build.fsx code</summary>\n",
```

Comment:
wrapping code tags in summary tags is not rendered correctly by nbconvert, i would ideally suggest to not use inline html in general

![image](https://user-images.githubusercontent.com/21338071/193456115-07029e66-11fd-41a3-b2de-69193d730268.png)

This applies to all of your codetags hidden like this, with the additional issue of html being rendered directly this way.